extensions/ui/start.cmd.tsx

```diff
@@ -45,9 +129,12 @@ export class StartCmd implements Command {
       rebuild,
     });
 
-    // clear the user console before moving interactive.
-    this.clearConsole();
-
-    return <UIServerConsole uiServer={uiServer} />;
+    return (
+      <>
+        <ClearConsole />
```

Comment:
i'm not sure if the clear console is the right way to do it. you can create a component that have state and change the state. let ink do the console manipulation instead of doing it manually

Reply:
Didn't get it...
Code example?

Reply:
Do not clear on 'verbose'!